dll/cpl/desk/desktop.c

```diff
@@ -7,7 +7,24 @@
 
 #include "desk.h"
 
-#define FCIDM_DESKBROWSER_REFRESH    0xA220  /* From shresdef.h */
+#include <shlwapi.h>
+#include <shellapi.h>
+
+/* From shresdef.h */
+#define FCIDM_DESKBROWSER_REFRESH    0xA220
+
+#define IDS_TITLE_MYCOMP  30386
+#define IDS_TITLE_MYNET   30387
+#define IDS_TITLE_BIN_1   30388
+#define IDS_TITLE_BIN_0   30389
```

Comment:
Move the resource IDs to `resource.h`.

Reply:
These are external resources (from shell32.dll, not desk.cpl), are you sure they needs to be placed there anyway?

Reply:
Ah, don't move them.

Reply:
`/* From shell32.dll */`

Reply:
Yes, but there's a comment `/* From shresdef.h */` which is actual for all constants listed below it.